### PR TITLE
Create 2014-02-16 madelinecodingbat2.md and Nitrous Test Post

### LINKED ISSUE #312: Madeline's Heroku Post
#282

### LINKED ISSUE #313: Madeline's Nitrous Test Post
#282

### PR DESCRIPTION
**Pull request by madeline13**
fixes #312, fixes #313

### CODE REVIEW

File: _posts/2014-02-05-madelinetest.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+author: madeline
+title: Our house is a very very very fine house
+layout: post
+date: 2014-02-19
+---
+
+## That song is stuck in my head.
```
Comment (eah13) [inline _posts/2014-02-05-madelinetest.md:8]: Nice work with this command line post!

**Comment by eah13** [comment]
@madeline13 remember to add your github.io link to this pull request, up in the description. We forgot to do that together today.

**Comment by gerbal** [comment]
fixes #312, fixes #313

**Comment by eah13** [comment]
Merging this so you don't fall behind.

**Pull request merged**